src/gui/sdlmain.cpp

```diff
@@ -2317,9 +2317,7 @@ int main(int argc, char* argv[]) {
 		control->StartUp();
 		/* Shutdown everything */
 	} catch (char * error) {
-#if defined (WIN32)
-		sticky_keys(true);
-#endif
+		rcode = 1;
```

Comment:
This stick-key re-enabler is redundant because it's also re-enabled at the end of the function (after the catch statements).

Reply:
Nice one :)